app/views/shared/_og_meta_tags.html.erb

```diff
@@ -0,0 +1,16 @@
+<!-- Basic Meta Tags -->
+<meta name="title" content="<%= t('app.name') %>">
+<meta name="description" content="<%= t('app.description') %>">
+
+<!-- Open Graph Meta Tags -->
+<meta property="og:title" content="<%= t('app.name') %>">
+<meta property="og:description" content="<%= t('app.description') %>">
+<meta property="og:image" content="https://mdbcdn.b-cdn.net/img/new/slides/041.webp">
```

Comment:
You should change this url to the Maxiambiental logo, or other related image. Here was just an example.